connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -6975,6 +6975,71 @@ object functions {
    */
   def parse_json(json: Column): Column = Column.fn("parse_json", json)
 
+  /**
+   * Check if a variant value is a variant null. Returns true if and only if the input is a
+   * variant null and false otherwise (including in the case of SQL NULL).
+   *
+   * @param v
+   *   a variant column.
+   * @group variant_funcs
```

Comment:
it seems we don't have this `variant_funcs` group

Reply:
Actually, we do have this group: https://github.com/apache/spark/blob/fe47edece059e9189d8500b3c9b3881b44678785/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/variant/variantExpressions.scala#L54. `parse_json` in `functions.py/scala` was incorrectly marked `json_funcs`, I also fix it by the way.

Reply:
We should add relevant test cases in PlanGenerationTestSuite for `connect`

Reply:
Added. Please take a look.

Reply:
@chenhao-db I mean group `variant_funcs` is missing [here](https://github.com/apache/spark/blob/9f34b8eca2f3578367352ec28532e29fe3a91864/connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala#L58-L81)